chart-app/src/components/ChartApp.vue

```diff
@@ -3,28 +3,29 @@
         <h1>{{msg}}</h1>
 
         <!--    This div is the container for the chart component-->
-        <div class="chart-container">
+        <div id="chart-container" class="chart-container" hidden>
             <!-- Using Chart.js in a Vue.js Application-->
             <!-- Step 2. Modify App.vue to use the new Chart component-->
             <SampleChart />
+            <p>{{key}}</p>
         </div>
 
         <!--    This div is the container for the buttons-->
         <div class="controls-container">
 
             <div class="dropdown-container">
-                <select id="chart-select" name="charts">
-                    <option value="" hidden disabled selected>--Please select a chart--</option>
-                    <option v-for="(chart, name) in charts" :key="name" :value="name">{{chart.label}}</option>
+                <select id="chart-select" v-model="key" @change="revealChart($event)">
```

Comment:
```suggestion
                <select id="chart-select" v-model="activeChart">
```
Since `v-model` is already updating a variable, you generally want to base any 'reaction' on this variable - rather than calling another function. You can see how I've done this with a conditional class above and the computed value below.

Also I change the variable name because `key` has a pre-defined meaning in the context of a select menu so it could cause confusion, and it's best to be super literal.